README.md

```diff
@@ -41,18 +41,17 @@ Every time a Docker object changes, the plugin updates the Caddyfile and trigger
   * [Connecting to Docker Host](#connecting-to-docker-host)
   * [Volumes](#volumes)
   * [Trying it](#trying-it)
-    + [With docker-compose file](#with-docker-compose-file)
+    + [With docker compose file](#with-docker-compose-file)
     + [With run commands](#with-run-commands)
   * [Building it](#building-it)
 
-## Basic usage example, using docker-compose
+## Basic usage example, using docker compose
 ```shell
 $ docker network create caddy
 ```
 
-`caddy/docker-compose.yml`
+`caddy/compose.yml`
```

Comment:
Docker Compose v2 switched convention of the extension to prefer / advise using `.yaml` instead of `.yml` (_both are still supported, just part of the convention change_).

```suggestion
`caddy/compose.yaml`
```